helm-service/controller/configuration_changer.go

```diff
@@ -410,6 +413,23 @@ func (c *ConfigurationChanger) changeCanary(e *keptnevents.ConfigurationChangeEv
 	return nil
 }
 
+func (c *ConfigurationChanger) scaleDownCanaryDeployment(e *keptnevents.ConfigurationChangeEventData) error {
+	client, err := keptnutils.GetClientset(true)
+	if err != nil {
+		return err
+	}
+	deployment, err := client.AppsV1().Deployments(e.Project+"-"+e.Stage).Get(e.Service, v1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	deployment.Spec.Replicas = int32Ptr(0)
+	_, err = client.AppsV1().Deployments(e.Project + "-" + e.Stage).Update(deployment)
```

Comment:
I don't have a clue what the API provides here, but I was wondering:
we already have `deployment`. Is it not possible to update `deployment` directly?
if not, would it be possible to store  `client.AppsV1().Deployments(e.Project+"-"+e.Stage)` in a variable?